src/assets/scss/views/shared/_chart.scss

```diff
@@ -97,16 +97,16 @@
 .chart-container-statistics {
   position: relative;
   margin: auto;
-  width: 75vw;
-  height: 70vh;
-  @media (max-width: 1280px) {
-    width: 90vw;
-    height: 50vh;
-  }
+  width: calc((100vw - 260px) * 0.95);
+  height: calc((100vw - 260px) * 0.5);
   @media (max-width: 991px) {
```

Comment:
You can use bootstrap right in the HTML:
https://getbootstrap.com/docs/4.4/utilities/sizing/

Reply:
sounds like sizing in the html won't handle the "calc" as I need to substract the navbar width.

Reply:
Hum, sounds like more a 'hack' then :)
But if we have no other options...

Reply:
Yes it's a hack thing as viewport width take in account the total viewport that's why we had some missing values depending on the size of the window. 
What we need is the "usable" viewport width for the chart (means without the nav bar) please tell if I need to show you